Plugins/NativeEngine/Source/NativeEngine.cpp

```diff
@@ -1530,4 +1532,13 @@ namespace Babylon
             function();
         });
     }
+
+    void NativeEngine::SetHardwareScalingLevel(const Napi::CallbackInfo& info){
```

Comment:
nit: Style. I forget how to run it in VS, but I think the clang formatter should be able to format this for you.

Reply:
oopsie. beautify in vsCode seems to follow the clang formatting style :)